pkg/kv/dist_sender.go

```diff
@@ -59,6 +60,39 @@ const (
 	defaultSenderConcurrency = 500
 )
 
+var (
+	// metaDistSenderTotals identify the total rpc sends of DistSender.
+	metaDistSenderTotals = metric.Metadata{Name: "distsender.rpc.totals"}
```

Comment:
still missing the help strings in the metadata.
we haven't ported all metrics to have those, but all new ones definitely should.
for example, from `pkg/storage/metrics.go`:
```
  metaRaftRcvdSnap = metric.Metadata{
    Name: "raft.rcvd.snap",
    Help: "Total number of MsgSnap messages received by this store",
  }
```

A description of the relationship between the metrics would also be helpful.